Bluetooth: Mesh: Added Ambient Light Sensor (#11127)
nrfconnect/sdk-nrf: samples/bluetooth/mesh/sensor_server/README.rst

   :depth: 2

The Bluetooth® mesh sensor sample demonstrates how to set up a basic mesh Sensor Server model application that provides sensor data to one :ref:`bt_mesh_sensor_cli_readme` model.

> Five different sensor types are used to showcase different ways for the server to publish data.

Author: No there is five sensors, but they also have 4 settings. If not then this wrong before to. But there is saying sensor types, so maybe it is correct with 8, as the last settings is same type as the sensor type.

> Five different sensor types are used to showcase different ways for the server to publish data.

Reviewer: I liked the way it was before, where this is specifically the number of sensors returned by the Sensor Descriptor Get. I was never really a fan of the setting types being listed in the list on line 43 either. Meaning: keep this as 5. I'm not going to delay this about the list below, I think I tried that before and the result was that they were kept that way.

> Five different sensor types are used to showcase different ways for the server to publish data.

Author: I could make this more visible by indenting the settings and then showing the five sensors.

> Five different sensor types are used to showcase different ways for the server to publish data.

Reviewer 2: I at least liked the new list structure better. It becomes more clear what the five sensor types are and what is considered a setting.
We should probably align and have the same in readme file for the sensor client.

* :c:var:`bt_mesh_sensor_dev_op_temp_range_spec` - Used as a setting for the :c:var:`bt_mesh_sensor_present_dev_op_temp` sensor type to set the range of reported temperatures.
* :c:var:`bt_mesh_sensor_rel_runtime_in_a_dev_op_temp_range` - Periodically requested by the client.
* :c:var:`bt_mesh_sensor_presence_detected` - Published when a button is pressed on the server.
* :c:var:`bt_mesh_sensor_time_since_presence_detected` - Periodically requested by the client and published by the server according to its publishing period (see :ref:`bluetooth_mesh_sensor_server_conf_models`).
* :c:var:`bt_mesh_sensor_motion_threshold` - Used as a setting for the :c:var:`bt_mesh_sensor_presence_detected` sensor type to set the time (0-10 seconds) before the presence is detected.
* :c:var:`bt_mesh_sensor_present_amb_light_level` - Periodically requested by the client.
* :c:var:`bt_mesh_sensor_gain` - Used as a setting for the :c:var:`bt_mesh_sensor_present_amb_light_level` sensor type to set the gain the ambient light sensor value is multiplied with.

> * :c:var:`bt_mesh_sensor_present_amb_light_level` - Used as a setting for the :c:var:`bt_mesh_sensor_present_amb_light_level` sensor type to calculate sensor gain based on measured reference ambient light level. This value does only have a set command.

Author: No the bt_mesh_sensor_present_amb_light_level is used as a setting to a bt_mesh_sensor_present_amb_light_level sensor type.

> The following Bluetooth mesh sensor types are used in this sample:

Reviewer 2: ```suggestion
The following Bluetooth mesh sensor types, and their settings, are used in this sample:
```

> Button 1:
>    Simulates presence detected. The time the button need to be pressed, before the presence is detected, depends on the motion threshold. The motion threshold has five steps from 0% (representing 0 seconds) to 100% (representing 10 seconds) separated by 25%.
> 
> Button 2:
>    Simulates different ambient light sensor values. These dummy values represent a raw value coming from a ambient light sensor.

Reviewer 2: ```suggestion
Button 1:
   Simulates presence detected.
   For how long the button has to be pressed before the presence is detected depends on the motion threshold.
   The motion threshold is defined from 0 % (representing 0 seconds) to 100 % (representing 10 seconds) in 25 %-steps.

Button 2:
   Simulates different ambient light sensor values.
   These dummy values represent raw values coming from an ambient light sensor.
```
